src/main/java/com/dnd/runus/infrastructure/persistence/jooq/running/JooqRunningRecordRepository.java

```diff
@@ -27,4 +39,39 @@ public int findTotalDistanceMeterByMemberId(long memberId, OffsetDateTime startD
         }
         return 0;
     }
+
+    public List<RunningRecordWeeklySummary> findWeeklyDistanceSummaryMeter(
+            long memberId, OffsetDateTime startWeekDate) {
```

Comment:
다른 repository 함수처럼 날짜 구간의 시작과 끝은 service 계층에서 관리하면 좋을 것 같아요
만약 그러면 함수 이름도 달라져야 할 것 같아요

```suggestion
            long memberId, OffsetDateTime startDate, OffsetDateTime endDate) {
```

Reply:
날짜 구간의 시작과 끝은 service 계층에서 관리하는게 좋을 것 같네요!
 그러면 함수 네이밍을 어떻게 해야할지 고민이에요. 좋은 네이밍이 떠오르지 않네요ㅠㅠㅠㅠㅠ

Reply:
`findMeterSumRange`? 와 같은 이름이 떠오르는 데, 잘 안 떠오르네요!